src/meson.build

```diff
@@ -0,0 +1,85 @@
+gnotime_cfg_data = configuration_data()
+gnotime_cfg_data.set_quoted('DATADIR', get_option('prefix') / get_option('datadir') / 'gnotime')
```

Comment:
This macro is never used, as far as I can tell. There are three macros set to the same directory location, the other two are used though.